governance/voter-weight-addin/program/src/processor.rs

```diff
@@ -29,19 +30,30 @@ pub fn process_instruction(
     msg!("GOVERNANCE-VOTER-WEIGHT-INSTRUCTION: {:?}", instruction);
 
     match instruction {
-        VoterWeightAddinInstruction::Revise {} => Ok(()),
-        VoterWeightAddinInstruction::Deposit { amount } => {
-            process_deposit(program_id, accounts, amount)
-        }
-        VoterWeightAddinInstruction::Withdraw {} => Ok(()),
+        VoterWeightAddinInstruction::SetupVoterWeightRecord {
+            voter_weight,
+            voter_weight_expiry,
+            weight_action,
+            weight_action_target,
+        } => process_deposit(
+            program_id,
+            accounts,
+            voter_weight,
+            voter_weight_expiry,
+            weight_action,
+            weight_action_target,
+        ),
     }
 }
 
 /// Processes Deposit instruction
 pub fn process_deposit(
```

Comment:
I got a bit confused here -- how about `process_setup_voter_weight_record` or `process_resolve_voter_weight`?

Reply:
correct, I missed that, renamed now